frontends/search/src/components/ResultsPage.tsx

```diff
@@ -212,9 +212,16 @@ const ResultsPage = (props: ResultsPageProps) => {
           : (sort_by = undefined);
       }
 
+      const query =
+        props.search.debounced.multiQueries.length > 0
+          ? props.search.debounced.multiQueries
+              .map((q) => [q.query, q.weight])
+              .filter((q) => q[0] != "")
+          : props.search.debounced.query;
+
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const requestBody: any = {
-        query: props.search.debounced.query,
```

Comment:
you still would want the ResultsPage to use the debounced query so that it doesn't send double requests.

Reply:
Above it's still using debounce, so going to resolve for now.